bittensor/_dataloader/dataloader_impl.py

```diff
@@ -36,19 +36,22 @@ def __init__(self):
         # TODO (shibshib): Find a proper way to set this as config instead of hardcoding it.
         # More dataset hashes can be added as we add directories for other modalities.
         self.genesis_text_dataset_hash = "QmXwfPoh2QFYqC6cYcW8kzyd9ruFfhnUi2kVBkdhawjUzj"
+        self.wikitext_text_dataset_hash = 'QmRjFNn3XpYMycVzTE4YcVcxk45vNZcTAjKmtEqPLKwAWd'
+        self.test_text_dataset_hash = 'QmRhWSMPQzTiWcdGYy8vpRMxSxCAKDJBaXvmum4fjkF7cJ'
+        self.validation_text_dataset_hash = 'QmQnE8wBmxKgNteFkZ1RAdZFit16iSeHwX6zSpYfwFmAuG'
 
         # Used to retrieve directory contentx
-        self.dag_get = 'https://ipfs.infura.io:5001/api/v0/dag/get'
+        self.dag_get = 'https://gateway.pinata.cloud/api/v0/dag/get'
         # Used to retrieve file contents
-        self.file_cat = 'https://ipfs.infura.io:5001/api/v0/cat'
+        self.file_cat = 'https://gateway.pinata.cloud/api/v0/cat'
 
         # Used when current corpus has been exhausted
         self.refresh_corpus = False
 
     @staticmethod
     def requests_retry_session(
-            retries=3,
-            backoff_factor=0.3,
+            retries=5,
```

Comment:
Any reason for 5 instead of 3?

Reply:
Slightly longer load times to account for possible traffic in IPFS, had a few issues with IPFS being too slow and timing out.